_docs/taxon-constraints.md

```diff
@@ -2,7 +2,7 @@
 title: Taxon constraints in the Gene Ontology
 permalink: /docs/taxon-constraints/
 redirect_from: 
-- /cgi-bin/references.cgi#GO_REF:0000056
+- /cgi-bin/references.cgi
```

Comment:
Hm, this does not seem to work?

Reply:
Yeah, I also noticed that the redirect doesn't seem to work on the production site currently -- with or without the hash. Other pages with `redirect_from` items seem to work okay. I'm not sure what's special about this one. Nonetheless it's definitely orthogonal to these changes. So maybe a new issue to investigate is warranted?

Reply:
Fair enough--I couldn't find a way to make it work. As we've had no complaint (that I know of), it's more an oddity than anything else.